platforms/openid-connect/README.md

```diff
@@ -0,0 +1,110 @@
+## Unleash Community Docker Image with OpenID Connect Support
+
+**Useful links:**
+
+- [Docker image on dockerhub](https://hub.docker.com/r/unleashorg/unleash-community-openid-connect)
+- [passport-openid-connect](https://www.passportjs.org/packages/passport-openidconnect/)
+
+For an example usage see the example below.
+You can also visit the [passport](https://www.passportjs.org/docs/) and [openid-connect](https://www.passportjs.org/packages/passport-openidconnect/) documentations for further information.
+
+## Example
+
+This shows the usage of the OpenIdConnectStrategy with a keycloak realm as IDP.
+It authenticates the user and checks some custom roles to give the user some initial permissions. 
+It also updates the role in case it changed, so you can completely control unleash permissions through roles in keycloak.
+You need to configure your client accordingly, so it returns roles as a `resource_access` object.
+
+If you only need the authentication, just remove that part.
+
+```javascript
+const unleash = require('unleash-server')
+
+function openIdConnect(app, config, services) {
+    const {userService} = services;
+
+    passport.use(new OpenIdConnectStrategy({
```

Comment:
```suggestion
    passport.use('openidconnect', new OpenIDConnectStrategy({
```

We need to name the passport strategy in order to use it, later on in the code

Reply:
This comes from the "official" example from the [passport documentation](https://www.passportjs.org/packages/passport-openidconnect/). 

The name is read from the strategies' `name` property if no `name` was given in the `passport.use` method, which is also `openidconnect`. 
But I see your point. For better clarity on where the strategy name is coming from we can define it here. :+1: